presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -13,9 +14,11 @@ class AnonymizerEngine:
     """
 
     logger = logging.getLogger("presidio-anonymizer")
+    builtin_anonymizers = {"mask": Mask, "fpe": FPE, "replace": Replace, "hash": Hash,
```

Comment:
can't we replave this hard coded list with looking for all the sub classes of Anonymizer class? this will allow adding more trasformation types in the future without needing to update this list

Reply:
I suggest to start with this implementation and add a new user story to improve it. @navalev

Reply:
agree. Please add a TODO with the user story. I'm approving this PR and we will continue the work in a seperate one

Reply:
@navalev  USER STORY 2767